ovh/types_vrack.go

```diff
@@ -53,16 +54,45 @@ func (v VrackIp) ToMap() map[string]interface{} {
 	obj["gateway"] = v.Gateway
 	obj["ip"] = v.Ip
 	obj["zone"] = v.Zone
+	obj["region"] = v.Region
 
 	return obj
 }
 
+// IPv4 Block Payload
 type VrackIpCreateOpts struct {
-	Block string `json:"block"`
+	Block  string  `json:"block"`
+	Region *string `json:"region,omitempty"`
 }
 
 func (opts *VrackIpCreateOpts) FromResource(d *schema.ResourceData) *VrackIpCreateOpts {
 	opts.Block = d.Get("block").(string)
+
+	if region := helpers.GetNilStringPointerFromData(d, "region"); region != nil {
```

Comment:
you could just do `opts.Region = helpers.GetNilStringPointerFromData(d, "region")`